js/app.js

```diff
@@ -1 +1,202 @@
+const playbtn = document.getElementById('play-btn')
+const hitbtn = document.getElementById('hit-btn')
+const standbtn = document.getElementById('stand-btn')
+const statusMsg = document.querySelector('.game-status')
+const playerInfo = document.querySelector('.playersInfo')
+const dealerInfo = document.querySelector('.dealersInfo')
+const dealerTotal = document.getElementById('dealerTotal')
+const playerTotal = document.getElementById('playerTotal')
+
+
+/*----------------------------- Event Listeners ----------------------------*/
+playbtn.addEventListener('click', playGame)
+hitbtn.addEventListener('click', hit)
+standbtn.addEventListener('click', stand)
+
+
+/*-------------------------------- Functions --------------------------------*/
+
+init ()
+
+function init() {
+  cardDeck = ["dA","dQ","dK","dJ","d10","d09","d08","d07","d06","d05","d04","d03","d02","hA","hQ","hK","hJ","h10","h09","h08","h07","h06","h05","h04","h03","h02","cA","cQ","cK","cJ","c10","c09","c08","c07","c06","c05","c04","c03","c02","sA","sQ","sK","sJ","s10","s09","s08","s07","s06","s05","s04","s03","s02"]
+}
+
+
+function playGame() {
+  statusMsg.textContent = ''
+  gameWinner = false
+  init()
+  shuffleCards(cardDeck)
+  playerHand = []
+  dealerHand = []
+  dealCards()
+  hitbtn.disabled = false
+  standbtn.disabled = false
+  renderCards()
+  checkBlackjack(countHands(playerHand), countHands(dealerHand))
+}
+
+
+function shuffleCards(arr){
+  for(var i =arr.length-1 ; i>0 ;i--){
+      var j = Math.floor( Math.random() * (i + 1) );
+      [arr[i],arr[j]]=[arr[j],arr[i]]; 
+  }
+}
+
+function dealCards() {
+  playerHand.push(cardDeck.pop())
+  dealerHand.push(cardDeck.pop())
+  playerHand.push(cardDeck.pop())
+  dealerHand.push(cardDeck.pop())
+  console.log(playerHand, dealerHand)
+  console.log(cardDeck)
+}
+
+
+function renderCards() {
+  playerInfo.innerHTML = ""
+  dealerInfo.innerHTML = ""
+  for(i= 0; i< playerHand.length; i++) {
+    let card = document.createElement("div")
+    card.className = `card xlarge ${playerHand[i]}`
+    playerInfo.appendChild(card)
+  }
+  for(i = 0; i < dealerHand.length; i++) {
+    let card = document.createElement("div")
+    card.className = `card xlarge ${dealerHand[i]}`
+    dealerInfo.appendChild(card)
+  }
+  hitbtn.style.visibility = "visible"
+  standbtn.style.visibility = "visible"
+}
+
+
+function cardValues(card) {
+  let value
+  
+  if (ace.includes(card)) {
+    value = 11
+  }
+  if (pictureCard.includes(card)) {
+    value = 10
+  }
+  if (nineSuits.includes(card)) {
+    value = 9
+  }
+  if (eightSuits.includes(card)) {
+    value = 8
+  }
+  if (sevenSuits.includes(card)) {
+    value = 7
+  }
+  if (sixSuits.includes(card)) {
+    value = 6
+  }
+  if (fiveSuits.includes(card)) {
+    value = 5
+  }
+  if (fourSuits.includes(card)) {
+    value = 4
+  }
+  if (threeSuits.includes(card)) {
+    value = 3
+  }
+  if (twoSuits.includes(card)) {
+    value = 2
+  }
+  return value 
+}
```

Comment:
Instead of doing all this work what if you used the info contained in the class name of the card itself?